source/testing/testing-components.md

```diff
@@ -129,182 +111,69 @@ jQuery triggers can be used to simulate user interaction and test that the title
-  assert.equal(finalTitle, 'Hello Ember World');
+  assert.equal(this.$('h2').text(), 'This is Magic', 'title changes after click');
 });
 ```
 
 ### Testing Actions
 
-Components often utilize the `sendAction()` method to send actions to other
-objects in your application.
+Components starting in Ember 2 utilize closure actions, which allow the component to directly invoke functions provided by components higher up in then tree.
 
-For example, imagine you have a comment form component that sends a specified
-`submit` action when the form is submitted, passing along the form's data:
+For example, imagine you have a comment form component that invokes a
+`submitComment` action when the form is submitted, passing along the form's data:
 
 > You can follow along by generating your own component with `ember generate
-> component comment-form`.
+> component comment-form --pod`.
 
-```app/components/comment-form.js
+```app/components/comment-form/component.js
 export default Ember.Component.extend({
-  body: null,
+  comment: '',
 
   actions: {
-    submit() {
-      var body = this.get('body');
-
-      this.sendAction('submit', { body: body });
+    submitComment() {
+      this.attrs.submitComment({ comment: this.get('comment') });
     }
   }
 });
 ```
 
-```app/templates/components/comment-form.hbs
-<form {{action "submit" on="submit"}}>
+```app/components/comment-form/template.hbs
+<form {{action "submitComment" on="submit"}}>
   <label>Comment:</label>
-  {{textarea value=body}}
+  {{textarea value=comment}}
 
-  <input type="submit" value="Submit">
+  <input type="submit" value="Submit"/>
 </form>
 ```
 
-You might use this component in your application like this:
-
-```handlebars
-{{comment-form submit="createComment"}}
-```
-
-Here's an example test that asserts that the specified `externalAction` action
-is sent when the component's internal `submit` action is triggered by making use
-of a test double (dummy object):
-
-```tests/unit/components/comment-form-test.js
-test('external action is triggered when form is submitted', function(assert) {
-  // This is important to make sure that the test fails if
-  // our assertion is never called
-  assert.expect(1);
-
-  // Create our test double
-  var targetObject = {
-    externalAction(attributes) {
-      // This assertion will be called when the action is triggered
-      assert.deepEqual(attributes, { body: 'You are not a wizard!' });
-    }
-  };
-
-  // Creates the component
-  var component = this.subject({
-    // Sets sample data
-    body: 'You are not a wizard!',
-
-    // Sets the targetObject to our test double
-    // (this is where sendAction will send its action)
-    targetObject: targetObject,
-
-    // Specifies which action to send to targetObject on submit
-    submit: 'externalAction'
-  });
-
-  // Renders the component to the page
-  this.render();
-
-  // Submits the form
-  this.$().find('input[type="submit"]').click();
-});
-```
-
-<!---
```

Comment:
Since this was already commented I went ahead and deleted.  I can work on bringing it back if we feel its important.  Now that we use integration tests I don't really think there's going to be much different about nested components that cause this section.  you'd just declare the components in the render template and assert the DOM